vobe.json

```diff
@@ -1,3 +1,6 @@
 {
-  "version": "0.0.2-alpha"
-}
\ No newline at end of file
+  "version": "0.0.3-alpha",
+  "authMail": "auth@vobe.io",
+  "mailUsername": "auth@vobe.io",
+  "mailPassword": "Haha, did you really think we would leak our password? ¯\_(ツ)_/¯"
```

Comment:
maybe we should move secrets to an entire new file.

Reply:
You're right, but maybe not in this branch.